src/app/component/MenuOverlay/MenuOverlay.style.scss

```diff
@@ -142,9 +142,8 @@
                     width: 100%;
                     height: 100%;
                     padding: calc(var(--header-height) + 20px) 1rem 1rem;
-                    box-shadow:
-                        inset 0 var(--header-height) var(--overlay-background),
-                        inset 0 calc(var(--header-height) + 1px) var(--overlay-divider-color);
+                    box-shadow: inset 0 var(--header-height) var(--overlay-background),
+                    inset 0 calc(var(--header-height) + 1px) var(--overlay-divider-color);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 24 spaces (indentation)](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4433567)